index.html

```diff
@@ -3,28 +3,142 @@
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Logoipsum</title>
     <link rel="stylesheet" href="style.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=DM+Sans:wght@700&family=Mulish&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Logoipsum</title>
 </head>
 <body>
-    <header>
-        <div class="wrapper">
-            <img src="image/Logo Color.jpg" alt="logo" class="logo_header" width="200" height="41">
-                <div class="container">
-                    <nav>
-                        <ul>
-                            <li>Home</li>
-                            <li>Features</li>
-                            <li>Pricing</li>
-                            <li>Blog</li>
-                        </ul>
-                    </nav>
+    <header class="header">
+        <div class="container">
+            <div class="header__inner">
+                <div class="logo_header">
+                    <img src="/TMS/image/logoheader.jpg" alt="logo__header">
                 </div>
+                <nav class="nav">
```

Comment:
the named classes are not quite correct, but this is not a problem, we will discuss this in the next lesson

Reply:
ok